src/theme/src/components/inputs.js

```diff
@@ -0,0 +1,85 @@
+import { minorScale } from '../../../scales'
+
+export default function getInputStyles(theme) {
```

Comment:
Can you have the filenames in this directory be singular? They should also match the name of the component where they are consumed so it's easy for us to track them down.